linode/manager
# chore: [M3-6363, M3-6371] - Remove unused `MonkeyWrench`, `PanelErrorBoundary` components

## Description 📝
Rather than migrate these unused components to MUIv5, we can remove them from the codebase.

I went through the rest of the `src/components` tickets, and it looks like all we have left are at least in use somewhere, even if they may not have styles to be migrated.

## Major Changes 🔄
**List highlighting major changes**
- Removes `MonkeyWrench` from `src/components`
- Removes `PanelErrorBoundary` from `src/components`

## How to test 🧪
Verify that these components had no current use in Cloud and are fine to delete.
